compute/cker/include/cker/operation/Fill.h

```diff
@@ -44,6 +44,26 @@ inline void Fill(const Shape &input_shape, int *input_data, const T value_data,
   }
 }
 
+template <typename T>
+inline void Fill(const Shape &input_shape, T input_data, const Shape &output_shape, T output_data)
+{
+  int input_size = input_shape.FlatSize();
+  int output_size = output_shape.FlatSize();
+
+  assert(output_size >= input_size);   // output_size must be greater than input size
+  assert(!(output_size % input_size)); // It must be 0
+
+  int dim = output_size / input_size;
+  for (int i = 0; i < dim; ++i)
+  {
+    for (int j = 0; j < input_size; ++j)
+    {
+      // output_data[i][j] = input_data[j];
+      output_data[i * input_size + j] = input_data[j];
```

Comment:
This PR works for the case : input [2,1,2] -> output [1,4,2] ? Hard to image :)

Reply:
Regarding [tf.broadcast's](https://www.tensorflow.org/api_docs/python/tf/broadcast_to) documents.
To use BroadcastTo op, Dimensions must be equal.

If input [2, 1, 2] -> ouput [x, 2, 1, 2] is ok.
If input[2, 1, 2] -> output [x, 1, 4, 2] is not ok.

Reply:
@chunseoklee said that this code works for "input[2, 1, 2] -> output [x, 1, 4, 2]" but " tf.broadcast's documents." said that those kinds of shapes are not supported.

@Hyunjun85 , what is your exact intention? In tensorflow lite source code, there is a shape checker for it. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/util/bcast.h#L183

Reply:
@semun-lee I had wrong understanding(logical error).
But now, I append that checking logic to resolve broadcasting issuse.